bedrock/firefox/templates/firefox/features/base.html

```diff
@@ -27,7 +27,7 @@
             {% block features_header_content %}{% endblock %}
           </div>
           <div class="header-download">
-            {{ download_firefox(dom_id='features-header-download', button_color='button-hollow-arrow dark') }}
+            {{ download_firefox(dom_id='features-header-download', button_color='button-arrow dark') }}
```

Comment:
Now that we're moving to a color that works well on both dark and light backgrounds, so we still need to `dark` and `light` class name variations?